MIGRATION.md

```diff
@@ -0,0 +1,90 @@
+# Migrating to the `cloudant-java-sdk` library
+This document is to assist in migrating from the `java-cloudant` (package: `com.cloudant.cloudant-client`) to the newly supported [`cloudant-java-sdk`](https://github.com/IBM/cloudant-java-sdk) (package: `com.ibm.cloud.cloudant`).
+
+## Initializing the client connection
+There are several ways to create a client connection in `cloudant-java-sdk`:
+1. [Environment variables](https://github.com/IBM/cloudant-java-sdk#authentication-with-environment-variables)
+2. [External configuration file](https://github.com/IBM/cloudant-java-sdk#authentication-with-external-configuration)
+3. [Programmatically](https://github.com/IBM/cloudant-java-sdk#programmatic-authentication)
+
+[See the README](https://github.com/IBM/cloudant-java-sdk#code-examples) for code examples on using environment variables.
+
+## Other differences
+1. Fetching the Database object first before performing additional operations is not required. For example, in the case of updating a document you would first call `getDocument` to fetch and then `putDocument` to update.
+2. Model classes are used instead of POJOs.
```

Comment:
What about adding an example with a common model?

Reply:
Do not you think the linked examples are enough? If not, could you help me with this?

Reply:
Yea maybe they are enough, although I'm wondering if we want to point to a common example in this statement e.g. `PUT /db/docid` using `Document` schema.  @ricellis wdyt?